crates/starknet-server/src/api/models/mod.rs

```diff
@@ -41,6 +41,23 @@ pub enum BlockId {
     Tag(Tag),
 }
 
+impl From<BlockId> for starknet_rs_core::types::BlockId {
+    fn from(block_id: BlockId) -> Self {
+        type ImportedBlockId = starknet_rs_core::types::BlockId;
```

Comment:
If imported can be removed

Reply:
Addressed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/50/commits/cbc1ccbc413b5744395a68f3c289a4f412b0afcd